app/views/needs/revisions/_new_note_form.html.erb

```diff
@@ -1,11 +1,16 @@
-<%= semantic_form_for(:note, url: notes_url) do |f| %>
-  <%= f.inputs do %>
+<%= form_for(:note, url: notes_url) do |f| %>
+  <fieldset>
```

Comment:
I think this needs `class="inputs"` from when I did it. Not sure if it adds any behaviour though, maybe for accessibility reasons 🤷‍♂️

Reply:
You've added it above as well so just assuming you forgot here 👍